code/modules/shuttle/spaceship_navigation_beacon.dm

```diff
@@ -38,9 +38,9 @@ obj/machinery/spaceship_navigation_beacon/emp_act()
 /obj/machinery/spaceship_navigation_beacon/multitool_act(mob/living/user, obj/item/multitool/I)
 	..()
 	if(panel_open)
-		var/new_name = "Beacon_[input("Enter the custom name for this beacon", "It be Beacon ..your input..") as text|null]"
+		var/new_name = stripped_input(usr, "Enter the custom codename for this beacon", "Beacon Rename")
 		if(new_name && Adjacent(user))
-			name = new_name
+			name = "Beacon [new_name]"
```

Comment:
```suggestion
			name = "Beacon_[new_name]"
```

Reply:
Eh, I personally think it looks better that way, as it really looks out of place with the underscore. Still, if maintainer thinks it should stay that way, I will accept this suggestion.

Reply:
Please do not make unrelated fixes in security fixes, especially not in critical security fixes like XSS.

This pr should have been merged by not, not held up because you want to play the "lets ask the maintainers" over a fucking grammer change.

Reply:
I understand, sorry about that, did not think it would any issue. Commited.